stdlib/builtins.pyi

```diff
@@ -1009,6 +1021,7 @@ class slice(Generic[_StartT_co, _StopT_co, _StepT_co]):
 
     def indices(self, len: SupportsIndex, /) -> tuple[int, int, int]: ...
 
+@disjoint_base
```

Comment:
pyright seems to be quite upset about a lot of tuple types in this PR, so I'm wondering if the addition of the decorator here specifically somehow breaks its special-casing for tuples